From ab0334082b5a3f1a604828d0907c995aec702d8c Mon Sep 17 00:00:00 2001
From: Jason Rogena <j.rogena@cgiar.org>
Date: Wed, 28 May 2014 09:16:58 +0300
Subject: [PATCH 171/213] Version 0.9.4: Sire's Country of origin

Changed the wording and position for the sire's country of origin
TextView, Spinner and AutoCompleteTextView in CowRegistration
activity
---
 .../mistro/farmer/CowRegistrationActivity.java     |  2 +
 .../main/res/layout/activity_cow_registration.xml  | 60 +++++++++++-----------
 MistroFarmer/src/main/res/values/strings_en.xml    |  5 +-
 3 files changed, 35 insertions(+), 32 deletions(-)

diff --git a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/CowRegistrationActivity.java b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/CowRegistrationActivity.java
index 02039e6..9218948 100644
--- a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/CowRegistrationActivity.java
+++ b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/CowRegistrationActivity.java
@@ -607,6 +607,8 @@ public class CowRegistrationActivity extends SherlockActivity implements MistroA
         dialogDeformityOkayB.setText(Locale.getStringInLocale("okay",this));
         ArrayAdapter countryArrayAdapter = ArrayAdapter.createFromResource(this,R.array.countries,android.R.layout.select_dialog_item);
         countryOfOriginACTV.setAdapter(countryArrayAdapter);
+        countryOfOriginACTV.setHint(Locale.getStringInLocale("specify_other_country", this));
+
         int totalDeformitySVHeight = 0;
         for (int i = 0, len = deformityArrayAdapter.getCount(); i < len; i++) {
             View listItem = deformityArrayAdapter.getView(i, null, deformityLV);
diff --git a/MistroFarmer/src/main/res/layout/activity_cow_registration.xml b/MistroFarmer/src/main/res/layout/activity_cow_registration.xml
index 2e80f5f..3024765 100644
--- a/MistroFarmer/src/main/res/layout/activity_cow_registration.xml
+++ b/MistroFarmer/src/main/res/layout/activity_cow_registration.xml
@@ -212,6 +212,36 @@
             android:textColor="@color/text_input_color"
             android:inputType="text"/>
         <TextView
+            android:id="@+id/common_countries_tv"
+            android:layout_height="wrap_content"
+            android:layout_width="wrap_content"
+            android:layout_marginTop="@dimen/activity_f_r_text_view_margin_top"
+            android:textSize="@dimen/normal_text_size"/>
+        <Spinner
+            android:id="@+id/common_countries_s"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/activity_f_r_edit_text_margin_top"
+            android:textColor="@color/text_input_color"
+            android:layout_marginLeft="@dimen/activity_f_r_edit_text_margin_left"/>
+        <TextView
+            android:id="@+id/country_of_origin_tv"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/activity_f_r_text_view_margin_top"
+            android:visibility="gone"
+            android:textSize="@dimen/normal_text_size"/>
+        <AutoCompleteTextView
+            android:id="@+id/country_of_origin_actv"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/activity_f_r_edit_text_margin_top"
+            android:layout_marginLeft="@dimen/activity_f_r_edit_text_margin_left"
+            android:textSize="@dimen/normal_text_size"
+            android:textColor="@color/text_input_color"
+            android:visibility="gone"
+            android:inputType="textCapWords|textPersonName"/>
+        <TextView
                 android:id="@+id/dam_tv"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
@@ -248,36 +278,6 @@
             android:textSize="@dimen/normal_text_size"
             android:textColor="@color/text_input_color"
             android:inputType="text"/>
-        <TextView
-            android:id="@+id/common_countries_tv"
-            android:layout_height="wrap_content"
-            android:layout_width="wrap_content"
-            android:layout_marginTop="@dimen/activity_f_r_text_view_margin_top"
-            android:textSize="@dimen/normal_text_size"/>
-        <Spinner
-            android:id="@+id/common_countries_s"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_marginTop="@dimen/activity_f_r_edit_text_margin_top"
-            android:textColor="@color/text_input_color"
-            android:layout_marginLeft="@dimen/activity_f_r_edit_text_margin_left"/>
-        <TextView
-            android:id="@+id/country_of_origin_tv"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_marginTop="@dimen/activity_f_r_text_view_margin_top"
-            android:visibility="gone"
-            android:textSize="@dimen/normal_text_size"/>
-        <AutoCompleteTextView
-            android:id="@+id/country_of_origin_actv"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_marginTop="@dimen/activity_f_r_edit_text_margin_top"
-            android:layout_marginLeft="@dimen/activity_f_r_edit_text_margin_left"
-            android:textSize="@dimen/normal_text_size"
-            android:textColor="@color/text_input_color"
-            android:visibility="gone"
-            android:inputType="textCapWords|textPersonName"/>
 
         <LinearLayout
             android:layout_width="match_parent"
diff --git a/MistroFarmer/src/main/res/values/strings_en.xml b/MistroFarmer/src/main/res/values/strings_en.xml
index 220ec9a..d335d28 100644
--- a/MistroFarmer/src/main/res/values/strings_en.xml
+++ b/MistroFarmer/src/main/res/values/strings_en.xml
@@ -221,7 +221,7 @@
     </string-array>
     <string name="gps_narrowing_down_on_loc_en">Please wait while GPS narrows down on your location</string>
     <string name="event_only_for_female_cattle_en">The selected event can only occur in female cattle</string>
-    <string name="other_countries_en">Other Countries</string>
+    <string name="other_countries_en">Extra Countries</string>
     <string name="no_times_milked_in_a_day_en">No. of times milked per day</string>
     <string name="calf_suckling_en">Calf suckling</string>
     <string-array name="calf_suckling_types_en">
@@ -262,7 +262,7 @@
     <string name="you_do_not_have_cows_en">You do not seem to have cows</string>
 
     <!-- Added after translation-->
-    <string name="ear_tag_no_or_name_en">You can enter the either the cow\'s name or ear tag number</string>
+    <string name="ear_tag_no_or_name_en">You can enter either the cow\'s name or ear tag number</string>
     <string name="age_or_dob_en">You can enter either the cow\'s age or date of birth</string>
     <string name="enter_age_or_dob_en">Please enter either the cow\'s age or date of birth</string>
     <string name="cow_too_young_en">The selected cow is too young</string>
@@ -275,4 +275,5 @@
     <string name="enter_breed_en">Please specify the other breed</string>
     <string name="breed_unknown_en">The breed you entered is unknown</string>
     <string name="extra_breed_en">Extra Breed</string>
+    <string name="specify_other_country_en">Specify other country here</string>
 </resources>
-- 
2.7.1

